lib/ecl/ecl_grid.cpp

```diff
@@ -5290,7 +5287,7 @@ ecl_grid_type * ecl_grid_get_lgr(const ecl_grid_type * main_grid, const char * _
   __assert_main_grid( main_grid );
   {
     char * lgr_name          = util_alloc_strip_copy( __lgr_name );
-    ecl_grid_type * lgr_grid = (ecl_grid_type*)hash_get(main_grid->LGR_hash , lgr_name);
+    ecl_grid_type * lgr_grid = main_grid->LGR_hash.at(lgr_name);
```

Comment:
Note that the `at` method throws if the element is not found. Is this the intended behaviour?
Alternatively you could use `find` which returns an iterator